# Example configuration:
# - velocity limits are around a 10% above the physical limits
# - acceleration limits are just low enough to avoid jerking

# Mandatory parameters
speed_lim_vx: 0.8
speed_lim_vy: 0.8
speed_lim_w: 5.4

accel_lim_vx: 0.3
accel_lim_vy: 0.3
accel_lim_w: 3.5

decel_lim_vx: 0.9
decel_lim_vy: 0.9
decel_lim_w: 10.5
# Optional parameters
frequency: 20.0
#decel_factor: 1.0

# Robot velocity feedback type:
#  0 - none
#  1 - odometry
#  2 - end robot commands
robot_feedback: 2

# optional 

# velocity_smoother_on: true #set this param on movel_redis/global_variables.yaml